src/solver/simulation/timeseries-numbers.cpp

```diff
@@ -703,17 +564,10 @@ void drawAndStoreTSnumbersForNOTintraModal(const array<bool, timeSeriesCount>& i
             {
                 auto& link = *(it->second);
                 const uint nbTimeSeries = link.directCapacities.timeSeries.width;
-                if (nbTimeSeries == 1)
```

Comment:
We don't need to assign **zero** to a TS number (in case associated size is **1**) : **zero** is the default value now